<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetCalibration</class>
 <widget class="QDialog" name="DialogSetCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Calibration</string>
  </property>
  <widget class="QWidget" name="layoutWidget_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Job ID : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_id_parent">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Chessboard Information</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>#X :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_chess_x_num">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_chess_y_num">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>#Y :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Square Size :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_chess_size">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chess_get">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chess_set">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget_calibration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>641</width>
     <height>131</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>120</height>
    </size>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get_calibration_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Calibration Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
